go2html.go

```diff
@@ -100,6 +100,12 @@ func (t *Template) Populate(rawReplacements map[string]interface{}) string {
 				result := fragment.template.Populate(nestedReplacement)
 				sb.WriteString(result)
 			}
+		case attrsInjection:
+			rawRepl, _ := rawReplacements[string(fragment)]
+			repl, _ := rawRepl.(map[string]string)
```

Comment:
let's panic for now if we can't assert the type of raw replacements

Reply:
Fixed, as said above in the first changes request.

Reply:
Done!